cli/command/service/rollback.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+func newRollbackCommand(dockerCli *command.DockerCli) *cobra.Command {
+	options := newServiceOptions()
+
+	cmd := &cobra.Command{
+		Use:     "rollback [OPTIONS]",
+		Aliases: []string{"rollback"},
+		Short:   "Rollback a service",
+		Args:    cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			flags := cmd.Flags()
+			flags.Bool("rollback", true, "")
+			flags.Bool("force", false, "Force update even if no changes require it")
+			addServiceFlags(flags, options, nil)
```

Comment:
I don't think we want to add all the service flags. We only want the force flag added.